public/i18n/de.json

```
@@ -30,5 +30,5 @@
   "map.load.error": "Missing value for 'map.load.error'",
   "map.not-initialized.error": "Missing value for 'map.not-initialized.error'",
   "asset.load.error": "Missing value for 'asset.load.error'",
-  "asset.parse.error": "Missing value for 'asset.parse.error'"
+  "asset.parse.error": "Fehler beim lesen der Datei '{{filename}}'"
```

What about French and Italian?

---

I'll translate it with deepl, but it also would not matter as all translations are not finished yet and this was just to demonstrate how arguments would be used. Also as it is a silent error the translation is never shown anywhere anyway. Just FYI: you could also reference other translation keys to translate specific error messages.